_docs/_latest/use/cli/inventory.md

```diff
@@ -5,21 +5,23 @@ order: 101
 
 # {{ page.title }}
 
-Forseti Inventory collects and stores information about your Google Cloud Platform (GCP)
-resources. The Inventory data is
-[transformed to a data model]({% link _docs/latest/concepts/models.md %}) Forseti Scanner and
-Explain use to perform their operations.
+Forseti Inventory collects and stores information about your Google
+Cloud Platform (GCP) resources. Inventory data is
+[transformed to a data model]({% link _docs/latest/concepts/models.md %})
+that Forseti Scanner and Explain use to perform their operations.
 
 ---
 
-## Running Inventory
+## Before you begin
```

Comment:
Same comment as before, let's make sure links to the old heading are updated.

Reply:
The old heading still exists below, I just added a "Before you begin" section here.

Reply:
Resolved.